base/utils/git.zsh

```diff
@@ -90,7 +90,7 @@ __zplug::utils::git::checkout()
     tags[dir]="$(__zplug::core::core::run_interfaces 'dir' "$repo")"
     tags[from]="$(__zplug::core::core::run_interfaces 'from' "$repo")"
 
-    do_not_checkout=( "gh-r" )
+    do_not_checkout=( "gh-r" "prezto" )
```

Comment:
I am seeing similar problems (`[zplug] ERROR: pathspec 'master' (at tag) did not match (modules/terminal)`) with oh-my-zsh. If I amend `do_not_checkout` by adding `"oh-my-zsh"`), it fixes it for me too. This seems like a hack though?